pkg/cmd/openshift.go

```diff
@@ -30,6 +34,7 @@ func openshiftCmd(vals *types.CmdValues) *cobra.Command {
 				fmt.Fprintln(os.Stderr, err.Error())
 				os.Exit(1)
```

Comment:
can we just return err? instead of printing and exiting? same thing as on line 22